mailbox/opensearch/src/main/java/org/apache/james/mailbox/opensearch/events/OpenSearchListeningMessageSearchIndex.java

```diff
@@ -131,7 +142,61 @@ public EnumSet<SearchCapabilities> getSupportedCapabilities(EnumSet<MessageCapab
             SearchCapabilities.AttachmentFileName,
             SearchCapabilities.PartialEmailMatch);
     }
-    
+
+    @Override
+    public Mono<Void> reactiveEvent(Event event) {
```

Comment:
Does it still make sense to extend `ListeningMessageSearchIndex` ?

Reply:
Yes
- Still re-use some methods from the abstract class
- For guice injection